client/src/components/ProductMain/ProductColumn/ProductDescription/ProductDescription.jsx

```diff
@@ -1,9 +1,20 @@
 function ProductDescription() {
-    return (
+  return (
+    <div>
+      <img
+        className="rounded pr-10 pt-14 w-full"
+        src="https://cdn2.unrealengine.com/egs-starwarsjedisurvivordeluxeedition-respawn-editions-g2-01-900x2254-5aeba6af2a69.jpg"
+        alt="Pic"
+      />
       <div>
-          <h1>ProductDescription</h1>
+        <div className="flex flex-row pr-10 pt-1">
+          <div className="flex flex-row justify-center items-center p-[10px] h-[35] bg-neutral-700 rounded w-full">
```

Comment:
```suggestion
          <div className="flex flex-row justify-center items-center py-2 w-full bg-neutral-700 rounded w-full">
```

this might give you the same effect you're looking for while being responsive.